docs/topics/compiler-reference.md

```diff
@@ -136,22 +123,61 @@ $ kotlinc @options/compiler.options hello.kt
 Enable usages of API that [requires opt-in](opt-in-requirements.md) with a requirement annotation with the given 
 fully qualified name.
 
-### -Xsuppress-warning
+### -Xrepl
+
+Activates the Kotlin REPL.
+
+```bash
+kotlinc -Xrepl
+```
+
+### Warning management
 
-Suppresses specific warnings [globally across the whole project](whatsnew21.md#global-warning-suppression), for example:
+#### -nowarn
+
+Suppress all warnings during compilation.
+
+#### -Werror
+
+Turn all warnings to compilation errors.
+
+#### -Wextra
+
+Enable [additional declaration, expression, and type compiler checks](whatsnew21.md#extra-compiler-checks) that
+emit warnings if true.
+
+#### -Xsuppress-warning
```

Comment:
Add a note that this flag is deprecated in favor of `-Xwarning-level`.
Also it makes sense to change the order of sections, so `-Xwarning-level` would go first

Reply:
Maybe we should remove all mentions of `-Xsuppress-warning` right now? since it was experimental, we don't need a separate announcement I think

Reply:
Nice, I like it